test/tsconfig.json

```diff
@@ -90,6 +90,7 @@
         "../src/rules/noInternalModuleRule.ts",
         "../src/rules/noInvalidThisRule.ts",
         "../src/rules/noNullKeywordRule.ts",
+        "../src/rules/noReferenceRule.ts",
```

Comment:
`src/tsconfig.json` should also have a similar change, running `grunt` should automatically update both files.